_assets/stylesheets/components/_hubspot-forms.scss

```diff
@@ -3,8 +3,8 @@ body .hs-button.primary,
 body input[type="submit"],
 body input[type="button"]:not(#getStartedBtn):not(#myStuffBtn) {
   color: #fff;
-  background-color: #3b6e8f;
-  border-color: #3b6e8f;
+  background-color: #253746;
+  border-color: #253746;
```

Comment:
Can we use a variable for this or add a new one? If not could you add an inline comment explaining why